jest.config.js

```diff
@@ -1,7 +1,10 @@
 /** @type {import('ts-jest/dist/types').InitialOptionsTsJest} */
+require('ts-node/register') // allows us to use typescript for setup script
+require('./scripts/loadProductionEnvVars.ts')
 
 module.exports = {
   projects: ['<rootDir>/jest.unit.config.js', '<rootDir>/jest.e2e.config.js'],
+  setupFiles: ['<rootDir>/scripts/loadProductionEnvVars.ts'],
```

Comment:
nit: Is it possible to do this only from `jest.e2e.config.js`?
It's only needed for e2e tests, right?